src/rtc/h5_prep.py

```diff
@@ -614,6 +617,16 @@ def get_metadata_dict(product_id: str,
              ALL_PRODUCTS,
              burst_in.range_pixel_spacing,
              'Slant range spacing of the source data in meters'],
+        'metadata/sourceData/azimuthResolution':  # 1.6.7
+            ['source_data_azimuth_resolution',
```

Comment:
```suggestion
            ['source_data_azimuth_resolution_in_meters',
```
To avoid confusion: az. resolution in seconds or meters